packages/docs/docusaurus.config.js

```diff
@@ -137,8 +136,7 @@ const config = {
         copyright: `Copyright © ${new Date().getFullYear()} DXOS.org`,
       },
       prism: {
-        theme: darkCodeTheme,
-        darkTheme: darkCodeTheme,
```

Comment:
I would keep this in here, doesn't hurt anything and we'll likely want to consider dark theme again down the road